securityscanutils/commands/utils.go

```diff
@@ -45,3 +45,11 @@ func GetImagesPerVersionFromFile(constraintsFile string) (map[string][]string, e
 
 	return imagesPerVersion, nil
 }
+
+func GetAdditionalContextFromFile(filename string) (string, error) {
```

Comment:
Given that this function is generic and could be re-used, do you think it would be valuable to define this in `fileutils`, something like:
```
func ReadFileString(filename string)(string, error)
```